src/jobs/hypothesis_eval/condor/utils.py

```diff
@@ -420,23 +389,30 @@ def sort_column(self):
 
     @property
     def fet_thresholds(self):
-        if self.job_type == "skim_with_gpt":
-            return {
-                "ab": self.job_specific_settings["skim"]["ab_fet_threshold"],
-                "bc": self.job_specific_settings["skim"]["bc_fet_threshold"]
-            }
-        elif self.is_km_with_gpt_direct_comp:
+        if self.is_skim_with_gpt:
             return {
-                "ab": self.job_specific_settings["km_with_gpt_direct_comp"]["ab_fet_threshold"]
+                "ab": self.job_specific_settings["ab_fet_threshold"],
+                "bc": self.job_specific_settings["bc_fet_threshold"]
             }
         else:
             return {
-                "ab": self.job_specific_settings["km_with_gpt"]["ab_fet_threshold"]
+                "ab": self.job_specific_settings["ab_fet_threshold"]
             }
 
     @property
     def censor_year(self):
-        return self.job_specific_settings.get("censor_year", 2024)
+        # Prefer upper bound if present; fallback to 2024 default
+        return self.job_specific_settings.get("censor_year_upper", 2025)
```

Comment:
hardcoded 2025 is going to cause huge problems in the future (2026 onwards). I'm not that familiar with the hyp eval job, if there is no censor year provided does it skip censoring? or does it crash?